nixos/modules/services/web-apps/peering-manager.nix

```
@@ -171,9 +199,18 @@ in {
       '' + lib.optionalString (cfg.peeringdbApiKeyFile != null) ''
         with open("${cfg.peeringdbApiKeyFile}", "r") as file:
           PEERINGDB_API_KEY = file.readline()
+      '' + lib.optionalString (cfg.enableOidc && cfg.oidcSecretFile != null) ''
+        with open("${cfg.oidcSecretFile}", "r") as file:
+          OIDC_RP_CLIENT_SECRET = file.readline()
       '';
 
-      plugins = lib.mkIf cfg.enableLdap (ps: [ ps.django-auth-ldap ]);
+      plugins = lib.mkIf (cfg.enableLdap || cfg.enableOidc) (ps: [
+        (lib.mkIf cfg.enableLdap ps.django-auth-ldap)
+
+        (lib.mkIf cfg.enableOidc ps.mozilla-django-oidc)
+        (lib.mkIf cfg.enableOidc ps.pyopenssl)
+        (lib.mkIf cfg.enableOidc ps.josepy)
+      ]);
```

Let's squash this a little bit:
```suggestion
      plugins = lib.mkIf (cfg.enableLdap || cfg.enableOidc) (ps: 
        (lib.optionals cfg.enableLdap [ ps.django-auth-ldap ]) ++
        (lib.optionals cfg.enableOidc (with ps; [ mozilla-django-oidc pyopenssl josepy ]))
      );
```